real-main/app/models/card/manager.py

```diff
@@ -147,6 +147,23 @@ def on_user_subscription_level_change_update_card(self, user_id, new_item, old_i
         ):
             self.dynamo.delete_card(card_template.card_id)
 
+    def on_user_change_update_photo_card(self, user_id, new_item, old_item=None):
+        new_photo_post_id = new_item.get('photoPostId')
+        old_photo_post_id = (old_item or {}).get('photoPostId')
```

Comment:
Since we don't care about the value of the photo post id, and it would be a bummer to have a bug because of a `photoPostId` of `0` or `false`, I think it might make more sense to just keep track of a boolean rather than the actual post id value

```suggestion
        new_has_photo = 'photoPostId' in new_item
        old_has_photo = 'photoPostId' in (old_item or {})
```